cypress/support/commands.js

```diff
@@ -62,6 +62,22 @@ Cypress.Commands.add('nextcloudCreateUser', (user, password) => {
 	})
 })
 
+Cypress.Commands.add('nextcloudUpdateUser', (user, password, key, value) => {
```

Comment:
Nothing related to this PR, but maybe we could consider wrapping all those nice helpers into a small js library, so that they could be reused across apps when writing cypress tests.

Reply:
Created #2201 to address this.